StoreSpidLogs/__test__/index.test.ts

```diff
@@ -65,26 +94,23 @@ describe("StoreSpidLogs", () => {
       }
     };
     const blobItem = await index(mockedContext as any, aSpidMsgItem);
-    expect(blobItem).toEqual({ spidRequestResponse: aSpidBlobItem });
-  });
-
-  it("should store a SPID response published into the queue", async () => {
-    const mockedContext = {
-      bindings: {
-        spidMsgItem: aSpidMsgItem
-      },
-      done: jest.fn(),
-      log: {
-        error: jest.fn()
-      }
+    expect(blobItem).toHaveProperty("spidRequestResponse");
+    const blob = blobItem as IOutputBinding;
+    const reqRes = JSON.parse(
+      decryptWithRsaPrivateKey(blob.spidRequestResponse)
+    );
+    const decryptedSpidBlobItem: SpidBlobItem = {
+      createdAt: aDate,
```

Comment:
It was necessary because, after decrypt `reqRes.createdAt` is not a Date, but a string. i.e: "2020-04-01 15:03:00.0Z" instead of 2020-04-01 15:03:00.0Z.
This can cause test failure on equality between `aSpidBlobItem` and `spidRequestResponse` payload